app/controllers/application_controller.rb

```diff
@@ -7,21 +7,29 @@ class ApplicationController < ActionController::Base
 
   before_filter :require_signin_permission!
 
-  helper_method :current_finder
+  helper_method :current_format
 
 private
 
+  def current_format
+    document_types.fetch(params.fetch(:document_type, nil), nil)
+  end
+
   def document_types
-    {
-      "cma-cases" => OpenStruct.new(
-        document_type: "cma_case",
-        title: "CMA Cases",
-      ),
+    data = {
+      "CMA Case" => CmaCase
     }
-  end
 
-  def current_finder
-    document_types.fetch(params.fetch(:document_type, nil), nil)
+    data.map do |k, v|
```

Comment:
This is sufficiently opaque that we probably need a comment showing the intended output.